diff -uNr rockdodger-0.6.0a/Makefile rockdodger-0.6.0a-new/Makefile
--- rockdodger-0.6.0a/Makefile	Sun Nov 17 23:56:12 2002
+++ rockdodger-0.6.0a-new/Makefile	Thu Jan 29 15:32:51 2004
@@ -15,14 +15,27 @@
 #   along with this program; if not, write to the Free Software
 #   Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA  02111-1307  USA
 
+DESTDIR=
+prefix=/usr/local
+bindir=$(prefix)/bin
+datadir=$(prefix)/share
+vardir=$(prefix)/var
+
 VERSION=0.6.0a
 PACKAGENAME=rockdodger
 NEWD=$(PACKAGENAME)-$(VERSION)
 TMP=/tmp
-OPTIONS=-DVERSION=\"$(VERSION)\" -DCOMPILEDATE=\"28/7/02\"
 SOUNDSOURCE=sound
 EXENAME=rockdodger
 
+pkgdatadir=$(datadir)/$(PACKAGENAME)
+pkgvardir=$(vardir)/lib/$(PACKAGENAME)
+
+DEFINES=-DVERSION=\"$(VERSION)\" -DCOMPILEDATE=\"28/7/02\" -DDATADIR=\"$(pkgdatadir)\" -DVARDIR=\"$(pkgvardir)\"
+CC=cc
+CFLAGS=-g
+LDFLAGS=
+
 #SOUNDLIBRARIES=-lvorbisfile -lvorbis -logg
 #SOUNDLIBRARIES=
 SOUNDLIBRARIES=-lSDL_mixer
@@ -32,10 +45,10 @@
 all:	$(EXENAME)
 
 .c.o:
-	cc -c -g $? `sdl-config --cflags` $(OPTIONS)
+	$(CC) -c $(CFLAGS) $? `sdl-config --cflags` $(DEFINES)
 
 $(EXENAME):	SFont.o $(SOUNDSOURCE).o main.o
-	cc -o $(EXENAME) $+ $(LIBRARIES)
+	$(CC) $(LDFLAGS) -o $(EXENAME) $+ $(LIBRARIES)
 
 clean:
 	rm *.o $(EXENAME)
@@ -49,12 +62,13 @@
 	mv $(TMP)/$(NEWD).tar.gz .
 
 install:	all
-	# rhk made the point the next line must be a copy, in case /usr/games is on a different partition
-	cp $(EXENAME) /usr/games
-	mkdir -p /usr/share/rockdodger/data
-	cp data/*.{bmp,png,wav,mod} /usr/share/rockdodger/data
-	chmod a+r /usr/share/rockdodger/data/*
-	chmod a+r /usr/share/rockdodger/data
-	touch /usr/share/rockdodger/.highscore
-	chmod a+rw /usr/share/rockdodger/.highscore
+	mkdir -p $(DESTDIR)$(bindir)
+	cp $(EXENAME) $(DESTDIR)$(bindir)
+	mkdir -p $(DESTDIR)$(pkgdatadir)
+	cp data/*.{bmp,png,wav,mod} $(DESTDIR)$(pkgdatadir)
+	chmod a+r $(DESTDIR)$(pkgdatadir)/*
+	chmod a+r $(DESTDIR)$(pkgdatadir)
+	mkdir -p $(DESTDIR)$(pkgvardir)
+	touch $(DESTDIR)$(pkgvardir)/.highscore
+	chmod a+rw $(DESTDIR)$(pkgvardir)/.highscore
 
diff -uNr rockdodger-0.6.0a/main.c rockdodger-0.6.0a-new/main.c
--- rockdodger-0.6.0a/main.c	Sun Nov 17 23:56:12 2002
+++ rockdodger-0.6.0a-new/main.c	Thu Jan 29 14:58:14 2004
@@ -452,7 +452,7 @@
     FILE *f;
     mode_t mask;
     mask = umask(0111);
-    if (f=fopen("/usr/share/rockdodger/.highscore",mode)) {
+    if (f=fopen(VARDIR "/.highscore",mode)) {
 	umask(mask);
 	return f;
     }
@@ -918,7 +918,7 @@
 	    }
 	}
 	else {
-	    data_dir = strdup("/usr/share/rockdodger/data");
+	    data_dir = strdup(DATADIR);
 	    if (missing(data_dir)) {
 		fprintf (stderr,"Cannot find data in %s\n", data_dir);
 		exit(-2);
