diff -durN rockdodger-0.6.orig/Makefile rockdodger-0.6/Makefile
--- rockdodger-0.6.orig/Makefile	Sun Jul 28 23:16:52 2002
+++ rockdodger-0.6/Makefile	Tue Aug  6 09:24:32 2002
@@ -15,11 +15,16 @@
 #   along with this program; if not, write to the Free Software
 #   Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA  02111-1307  USA
 
+DESTDIR=
+bindir=/usr/games
+datadir=/usr/share/rockdodger
+vardir=/var/lib/rockdodger
+
 VERSION=0.6
 PACKAGENAME=rockdodger
 NEWD=$(PACKAGENAME)-$(VERSION)
 TMP=/tmp
-OPTIONS=-DVERSION=\"$(VERSION)\" -DCOMPILEDATE=\"28/7/02\"
+OPTIONS=-DVERSION=\"$(VERSION)\" -DCOMPILEDATE=\"28/7/02\" -DDATADIR=\"$(datadir)\" -DVARDIR=\"$(vardir)\"
 SOUNDSOURCE=sound
 EXENAME=rockdodger
 
@@ -49,12 +54,13 @@
 	mv $(TMP)/$(NEWD).tar.gz .
 
 install:	all
-	# rhk made the point the next line must be a copy, in case /usr/games is on a different partition
-	cp $(EXENAME) /usr/games
-	mkdir -p /usr/share/rockdodger/data
-	cp data/*.{bmp,png,wav,mod} /usr/share/rockdodger/data
-	chmod a+r /usr/share/rockdodger/data/*
-	chmod a+r /usr/share/rockdodger/data
-	touch /usr/share/rockdodger/.highscore
-	chmod a+rw /usr/share/rockdodger/.highscore
+	mkdir -p $(DESTDIR)$(bindir)
+	cp $(EXENAME) $(DESTDIR)$(bindir)
+	mkdir -p $(DESTDIR)$(datadir)
+	cp data/*.{bmp,png,wav,mod} $(DESTDIR)$(datadir)
+	chmod a+r $(DESTDIR)$(datadir)/*
+	chmod a+r $(DESTDIR)$(datadir)
+	mkdir -p $(DESTDIR)$(vardir)
+	touch $(DESTDIR)$(vardir)/.highscore
+	chmod a+rw $(DESTDIR)$(vardir)/.highscore
 
diff -durN rockdodger-0.6.orig/main.c rockdodger-0.6/main.c
--- rockdodger-0.6.orig/main.c	Sun Jul 28 23:16:52 2002
+++ rockdodger-0.6/main.c	Tue Aug  6 09:24:20 2002
@@ -452,7 +452,7 @@
     FILE *f;
     mode_t mask;
     mask = umask(0111);
-    if (f=fopen("/usr/share/rockdodger/.highscore",mode)) {
+    if (f=fopen(VARDIR "/.highscore",mode)) {
 	umask(mask);
 	return f;
     }
@@ -918,7 +918,7 @@
 	    }
 	}
 	else {
-	    data_dir = strdup("/usr/share/rockdodger/data");
+	    data_dir = strdup(DATADIR);
 	    if (missing(data_dir)) {
 		fprintf (stderr,"Cannot find data in %s\n", data_dir);
 		exit(-2);
